cmd/fluent-manager/Dockerfile

```diff
@@ -1,5 +1,9 @@
 # Build the manager binary
-FROM --platform=$BUILDPLATFORM golang:1.20.3-alpine3.17 as builder
+FROM ghcr.io/oracle/oraclelinux:8-slim as builder
```

Comment:
Should the common base image be used?

Reply:
Flipped the base image to the common base image